### DISCUSSION TITLE
SpaCy 2.2.4+ Doesn't Install on ARM64 Architecture

### DISCUSSION THREAD

**Discussion by JeffMII** [explosion/spaCy#9617] [Help: Installation] [answered]
## How to reproduce the behaviour
`pip install spacy==2.2.4`
or
`pip install spacy`

## Your Environment

* Operating System: Ubuntu Server 20.10
* Python Version Used:  Python 3.8.10
* spaCy Version Used: 2.2.4
* Environment Information: ARM64

## Error
```
gcc: error: unrecognized command line option ‘-mavx2’
gcc: error: unrecognized command line option ‘-mfma’
gcc: error: unrecognized command line option ‘-mfpmath=sse’
```

I am working on an NLP Web API in Python. The project works fine on my Windows machine, but I am getting the above error when I try running it on my Rasperry Pi 4. These errors seem to come from 'blis', but the problem was supposedly fixed according to an issue report in 2019 https://github.com/explosion/spaCy/issues/3451. That doesn't seem to be the case however.

**Comment by polm** [comment, marked as answer]
We recently had [a similar question](https://github.com/explosion/spaCy/discussions/9367#discussioncomment-1428399) about a Jetson Nano, which I think has similar hardware. v3 should just work for CPU usage on ARM64, though you might need to use conda. Can you try conda, or is upgrading to v3 an option for you?

There are also several more recent threads about running on raspberry pis, like [this one](https://github.com/explosion/spaCy/discussions/7372#discussioncomment-455387), that may be able to help you.

**Comment by JeffMII** [comment]
Actually, I did try installing spacy==3.0.0, but it doesn't work either.
